docs/docs/contributor/server/others/best-practices.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Best Practices
+sidebar_position: 3
+sidebar_custom_props:
+  icon: TbChecklist
+---
+
+This document outlines the best practices you should follow when working on the backend.
+
+## Follow a modular approach
+
+We follow a modular approach for our backend, which is a fundamental principle when working with NestJS. Make sure you break down your code into reusable modules to maintain a clean and organized codebase. 
+Each module should encapsulate a particular feature or functionality and have a well-defined scope. This modular approach enables clear separation of concerns and removes unnecessary complexities.  
+
+Most of our modules are directly related to a model in the database. For instance, we have a module for users, which is directly related to the user model in the database. 
```

Comment:
Nit: I would not talk about this last section. I feel like this part will be less and less true the more we go toward a dynamic schema approach (with custom/standard objects etc) and we will have mostly "dynamic" modules not tied to a particular model.